crates/rome_js_formatter/src/context.rs

```diff
@@ -16,6 +17,28 @@ pub struct JsFormatContext {
     /// The comments of the nodes and tokens in the program.
     comments: Rc<JsComments>,
 
+    /// Stores the formatted content of one function body.
+    ///
+    /// Used during formatting of call arguments where function expressions and arrow function expressions
+    /// are formatted a second time if they are the first or last call argument.
+    ///
+    /// Caching the body in the call arguments formatting is important because it minimises the cases
+    /// where the algorithm is quadratic in case the function or arrow expression contains another
+    /// call expression with a function or call expression as first or last argument.
```

Comment:
```suggestion
    /// Caching the body in the call arguments formatting is important. It minimises the cases
    /// where the algorithm is quadratic, in case the function or arrow expression contains another
    /// call expression with a function or call expression as first or last argument.
```

Long phrase without punctuation... I'm without breath 😆

Reply:
Yeah sorry, the bad habit of a native german speaker.